types/src/auction/bid.rs

```diff
@@ -23,16 +23,22 @@ pub struct Bid {
     /// `Some` indicates locked funds for a specific era and an autowin status, and `None` case
     /// means that funds are unlocked and autowin status is removed.
     release_era: Option<EraId>,
+    /// Delegators
```

Comment:
Suggestion: `A collection that tracks delegators attached to this particular validator`

Based on description from PR:

> adds a delegators field to the Bid type to track the delegators attached to a particular validator.

Reply:
Addressed